pkg/controller/servicecatalog/provisionedproduct/setup.go

```diff
@@ -356,8 +357,6 @@ func (c *custom) provisioningParamsAreChanged(ctx context.Context, cfStackParams
 		// the desired state. Because on cloudformation side spaces are also trimmed
 		if cfv, ok := cfStackKeyValue[*v.Key]; ok && strings.TrimSpace(pointer.StringDeref(v.Value, "")) == cfv {
```

Comment:
could probably switch this to `!= cfv` and `return true, nil` here and eliminate the `else` for cleanliness

Reply:
You right, thanks!
Fixed.